Machines/aleph_modules

```diff
@@ -0,0 +1,16 @@
+# >>> conda initialize >>>
+# !! Contents within this block are managed by 'conda init' !!
+__conda_setup="$('/home/jedwards/anaconda2/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
```

Comment:
anyway to make this path non-user specific? perhaps. an env var instead?

Reply:
All the users on this system are using my conda install.